cooltools/__init__.py

```diff
@@ -11,7 +11,7 @@
 """
 import logging
 
-__version__ = "0.6.1"
+__version__ = "0.6.2"
```

Comment:
If there are public API changes and/or new features, you want to do a minor bump. Patch bumps are for bug fixes. See [semantic versioning](https://semver.org/). 

Note that we are still in beta (major 0), we don't worry about major version bumps until our API is stable.

```suggestion
__version__ = "0.7.0"
```

Reply:
Ah ha, thank you! I update it to 0.7.0.